docs/source/markdown/podman-ps.1.md

```diff
@@ -61,6 +61,8 @@ Valid filters are listed below:
 | pod        | [Pod] name or full or partial ID of pod                                          |
 | network    | [Network] name or full ID of network                                             |
 | until      | [DateTime] container created before the given duration or time.                  |
+| command    | [Command] the command the container is executing                                |
```

Comment:
This should likely highlight that it only matches argv[0] not the full command like some would expect.

Reply:
Yes, you are right, I did this commend in the code, however forgot to do it in man pages.